Calendar/BookingHelpers/BookingStaffHelpers.ps1

```diff
@@ -0,0 +1,54 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+function Get-MBPermissions {
+    param($Identity)
+    # Get the Mailbox Permissions
+    $MBPermissions = Get-MailboxPermission -Identity $Identity -ErrorAction SilentlyContinue
+    return $MBPermissions
```

Comment:
only calling this once, is there a reason why we need to have this as a function?